test cases/common/162 gcc func attributes/meson.build

```diff
@@ -0,0 +1,86 @@
+# Copyright © 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project('gcc func attributes', ['c', 'cpp'])
+
+# For msvc these will fail because msvc doesn't support __attribute__, for
+# Clang and GCC, they should pass.
+c = meson.get_compiler('c')
+cpp = meson.get_compiler('cpp')
+expected_result = c.get_id() != 'mscv'
+
+attributes = [
+  'alias',
```

Comment:
This file seems to have bad indenation everywhere (2 sp instead of 4).

Reply:
Unless I've completely misunderstood, meson files (not python files) use 2 space indentation.

Reply:
Correct.